src/common/buttons/button.module.scss

```diff
@@ -0,0 +1,21 @@
+@import '../../assets/stylesheets/colors.scss';
+
+.customButton {
+  background-color: $sky-blue;
+  height: 3em;
+  border-radius: 10px;
+  width: 40%;
+  color: white;
+  box-shadow: $sky-blue 0px 7px 29px 0px;
+}
+
+.customButton:hover {
+  background-color: $sky-blue-on-hover;
+  width: 41%;
+}
+
+.customButton:disabled {
+  background-color: $sky-blue-disabled;
+  width: 40%;
+  box-shadow: none;
+}
```

Comment:
newline

Reply:
Also general note: I like how you didn't add many layout settings here (a lot of people add margins to their buttons so it fits the page, but then when they're gonna reuse them they have trouble).

The `width` property set to `%` might be an issue later on, but you can refactor them then